react/features/participants-pane/components/web/FooterContextMenu.js

```diff
@@ -27,7 +27,11 @@ import {
     getParticipantCount,
     isEveryoneModerator
 } from '../../../base/participants';
-import { openSettingsDialog, SETTINGS_TABS } from '../../../settings';
+import {
+    openSettingsDialog,
+    SETTINGS_TABS,
```

Comment:
can you please put this one first?

Reply:
Happy to change.

Do you mean like this?:

```javascript
import {
    SETTINGS_TABS,
    openSettingsDialog,
    shouldShowModeratorSettings
} from '../../../settings';
```

Reply:
yep!

Reply:
Will update shortly. Thanks for reviewing!